app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::ApiController < ActionController::API
+  before_action :set_default_format
+  before_action :authenticate_token!
+
+   private
+
+     def set_default_format
+      request.format = :json
+    end
+
+    def authenticate_token!
+      payload = JsonWebToken.decode(auth_token)
+      @current_user = User.find(payload["sub"])
```

Comment:
why is sub used as a keyword here?

Reply:
Sir decode method is placing user.id in payload["sub"].